content/en/docs/reference/kubernetes-api/common-definitions/delete-options.md

```diff
@@ -71,7 +71,14 @@ DeleteOptions may be provided when deleting an API object.
 
   Whether and how garbage collection will be performed. Either this field or OrphanDependents may be set, but not both. The default policy is decided by the existing finalizer set in the metadata.finalizers and the resource-specific default policy. Acceptable values are: 'Orphan' - orphan the dependents; 'Background' - allow the garbage collector to delete the dependents in the background; 'Foreground' - a cascading policy that deletes all dependents in the foreground.
 
+- **ignoreStoreReadErrorWithClusterBreakingPotential** (boolean)
 
+By enabling this option, the user can perform an unsafe, and force delete of an undecryptable/corrupt resource. This option is behind a new ALPHA feature gate, and it is disabled by default. In order to use this option, the cluster operator must enable the feature by setting the command line option `--feature-gates=AllowUnsafeMalformedObjectDeletion=true`.
```

Comment:
I think we should mention the `unsafe-delete-ignore-read-errors` permission here since it is a requirement to use that option